CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Changelog
 
+## 0.2.0 (2023-02-24)
+
+- Added a reference table of Miden Assembly instructions to the Playground
+- Added ability to prove a program to the Playground
+- Refactored the front-end to use the Tailwind css framework
```

Comment:
nit: my fault from my previous comment, sorry, but CSS should probably be capitalized